app/_includes/landing_pages/structured_text.md

```diff
@@ -24,4 +24,12 @@
 </ul>
 {% endif %}
 
+{% if item.type == "toggle" %}
+
+<div>
+<summary>{{ item.title | markdown }}</summary>
+<p>{{ item.text | markdown }}</p>
+</div>
+{% endif %}
```

Comment:
@fabianrbz I tried to add those toggles (similar to the one on this page https://docs.konghq.com/konnect/getting-started/add-api/#main), but they aren't formatted correctly.

Reply:
@cloudjumpercat you need to wrap that in a `<details>` tag